keyboards/karn/info.json

```diff
@@ -0,0 +1,71 @@
+{
+    "manufacturer": "Robert Mills",
+    "keyboard_name": "karn",
+    "maintainer": "robcmills",
+    "bootloader": "stm32-dfu",
```

Comment:
```suggestion
```

Reply:
@zvecr Removing this causes the build to fail: https://github.com/qmk/qmk_firmware/runs/11340797021
Where should the bootloader be specified?

Reply:
Welp, looks like the subsequent commit to change the dev board fixed it? https://github.com/qmk/qmk_firmware/pull/19846/commits/a37bb6ff00516027e5d03303b66e730785b325dc
Anyways, looks like builds are passing now 👍